include/dts-functions.sh

```diff
@@ -1375,6 +1344,137 @@ show_menu() {
   else
     echo -e "${BLUE}**${YELLOW}     ${DES_KEYS_OPT})${BLUE} Load your DES keys${NORMAL}"
   fi
+  if [ -f "${DES_SUBMENU_JSON}" ]; then
+    echo -e "${BLUE}**${YELLOW}     ${DES_SUBMENU_OPT})${BLUE} Premium options${NORMAL}"
```

Comment:
@pietrushnic Czy w aktualnej nomenklaturze Dasharo mamy odpowiednią nazwę na takie menu z featureami typu provisioining trust roota?

Reply:
According to the new agreement, we call it generically "Dasharo Products"